src/Makevars.in

```diff
@@ -7,7 +7,7 @@ LIBNAME = libr_polars.a
 TARGET_DIR = $(CURDIR)/rust/target
 LIBDIR = $(TARGET_DIR)/$(TARGET)/$(RPOLARS_PROFILE)
 STATLIB = $(LIBDIR)/$(LIBNAME)
-PKG_LIBS = -L$(LIBDIR) -lr_polars
+PKG_LIBS = -L$(LIBDIR) -lr_polars -lrt
```

Comment:
It seems that we should pass `-lrt` to Linux only.
https://github.com/cran/arrow/blob/61e0a7ade8ec52e2d1463c982632c3038c1a51bb/tools/cpp/CMakeLists.txt#L976-L979

Could you update the configure script and change it to not pass anything to macOS? (If it's too difficult, I can do it myself)

Reply:
OK, I gave it a shot in https://github.com/pola-rs/r-polars/pull/528/commits/fd673340f38ce7490534b3120dc3a7655a77fbf8